ui/i18n.js

```diff
@@ -61,20 +61,24 @@ function removeContextMetadata(message) {
   return message;
 }
 
-export function __(message: string, tokens: { [string]: string }) {
+function __(message /*: string */, tokens /*: { [string]: string } */) {
   if (!message) {
     return '';
   }
 
+  const navLang = globalThis.navigator?.language || '';
+
   const language = localStorageAvailable
-    ? window.localStorage.getItem('language') || 'en'
-    : window.navigator.language.slice(0, 2) || 'en';
+    ? globalThis.localStorage.getItem('language') || 'en'
+    : navLang.slice(0, 2) || 'en';
```

Comment:
This would be `globalThis.navigator?.language?.slice(0, 2)`, but the installed version of flow does not like optional chaining in the same expression as a function call.